### PR TITLE
update rtp process record to be compatible with scm

### PR DESCRIPTION
**Pull request by mkolopanis**
@jsdillon noticed that RTP is failing now that we switched all the repos to `setuptools_scm`. I updated the `add_rtp_process_record` to treat everything like how pyuvdata was parsed before.

### CODE REVIEW

File: scripts/add_rtp_process_record.py [hunk 1]
```diff
@@ -6,15 +6,13 @@
...
 import warnings
+import importlib
 from pkg_resources import parse_version
 
 import numpy as np
 from astropy.time import Time
 
 import hera_mc.mc as mc
-import hera_qm
-import hera_opm
```
Comment (jsdillon) [inline scripts/add_rtp_process_record.py:16]: Unfortunately, hera_opm is not on setuptools-scm yet, so this will still break.

Comment (mkolopanis) [reply scripts/add_rtp_process_record.py:16]: we've merged a new hera_opm that uses setuptools scm

**Comment by mkolopanis** [comment]
testing on 2459740. RTP is still failing but not because of the process record

**Comment by jsdillon** [comment]
RTP failures are not M&C's problem, just something that happens when the array is switched to load/noise. I think this PR solves the issue from RTP's perspective

**Review by plaplant — APPROVED** [review]
Looks good, thanks @mkolopanis! Feel free to merge/install anytime.

**Pull request merged**